SpatialGDK/Source/SpatialGDKTests/SpatialGDKEditor/SpatialGDKEditorSchemaGenerator/SpatialGDKEditorSchemaGeneratorTest.cpp

```diff
@@ -318,14 +320,16 @@ class SchemaTestFixture
 	void EnableSpatialNetworking()
 	{
 		UGeneralProjectSettings* GeneralProjectSettings = GetMutableDefault<UGeneralProjectSettings>();
-		bCachedSpatialNetworking = GeneralProjectSettings->UsesSpatialNetworking();
-		GeneralProjectSettings->SetUsesSpatialNetworking(true);
+		bCachedSpatialNetworking = USpatialStatics::IsSpatialNetworkingEnabled();
```

Comment:
Same here and in the rest of the PR, I prefer the previous usage of `GetDefault<USpatialGDKSettings>()->bSpatialNetworking;`